modules/howtos/pages/working-with-collections.adoc

```diff
@@ -18,18 +17,12 @@ Something that previously had to be done with key-value references to different
 
 == Travel Sample app -- with Collections
 
-include::6.5@sdk:shared:partial$sample-application.adoc[tag=prereq]
-
-[WARNING]
-====
-Enabling Developer Preview should only be done on a development machine; 
-there is no upgrade path available from a DP-enabled Couchbase Server.
-====
+include::7.0@sdk:shared:partial$sample-application.adoc[tag=prereq]
 
 
 == Preparation
```

Comment:
I've simply deleted all the `Preparation` section in my tickets, as it seems pretty much out of date, I think we need to revisit what we put here for all SDKs, could review on Friday?

Reply:
Yeah I decided not to touch that for the moment, as we are going to change this page anyways.